e2e-tests/monitoring-2-0/run

```diff
@@ -49,6 +49,11 @@ apply_cluster $test_dir/conf/$cluster-rs0.yml
 desc 'check if all 3 Pods started'
 wait_for_running $cluster-rs0 3
 
+API_KEY=$(curl --insecure -X POST -H "Content-Type: application/json" -d '{"name":"operator", "role": "Admin"}' "https://admin:$ADMIN_PASSWORD@"$(get_service_endpoint monitoring-service)"/graph/api/auth/keys" | jq .key)
+kubectl_bin patch secret some-users --type merge --patch '{"stringData": {"PMM_SERVER_API_KEY": '$API_KEY'}}'
+sleep 5
+wait_for_running $cluster-rs0 3
+
```

Comment:
`ADMIN_PASSWORD=$(kubectl_bin exec monitoring-0 -- bash -c "printenv | grep ADMIN_PASSWORD | cut -d '=' -f2")
kubectl_bin exec monitoring-0 -- bash -c "grafana-cli --homepath=/usr/share/grafana --config=/etc/grafana/grafana.ini admin reset-admin-password $ADMIN_PASSWORD"`
We don't need these rows if we don't use password

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/975/commits/9f6faf82aa82aa30744c4626a4f0de0fabc42a7e